packages/shorebird_cli/test/src/executables/patch_executable_test.dart

```diff
@@ -125,5 +131,45 @@ void main() {
         ),
       );
     });
+
+    group('when is on windows and we got a -1073741515 exit code', () {
```

Comment:
I would split this into two groups:

1. When running on windows
2. When process exits with code ...

This way we can test that we handle the missing C++ runtime on windows and have a separate test where the exit code is set but the platform is not windows to ensure we only have this special error handling case for windows.

Reply:
Generally, any time you write "and" in a group definition, that's a sign you probably want two groups.